src/LPPL-1.3a.xml

```diff
@@ -5,15 +5,6 @@
       <crossRefs>
          <crossRef>http://www.latex-project.org/lppl/lppl-1-3a.txt</crossRef>
       </crossRefs>
-      <standardLicenseHeader> Copyright 
-    <alt match=".+" name="copyright">2003 M. Y. Name</alt> This work may be distributed and/or modified under the
-    conditions of the LaTeX Project Public License, either version 1.3 of this license or (at your option) any later
-    version. The latest version of this license is in http://www.latex-project.org/lppl.txt and version 1.3 or later
-    is part of all distributions of LaTeX version 2003/12/01 or later. This work has the LPPL maintenance status
-    "maintained". This Current Maintainer of this work is 
```

Comment:
I've marked the maintenance status with `<alt>` to accept the other statuses mentioned in the license text.  This also affected `LPPL-1.3c`.